integrations/docker/images/chip-build-vscode/Dockerfile

```diff
@@ -1,11 +1,19 @@
 ARG VERSION=latest
 FROM connectedhomeip/chip-build:${VERSION}
 FROM connectedhomeip/chip-build-nrf-platform:${VERSION} AS nrf
+FROM connectedhomeip/chip-build-efr32:${VERSION} AS efr32
+FROM connectedhomeip/chip-build-android:${VERSION} AS android
 FROM connectedhomeip/chip-build-esp32:${VERSION}
 FROM connectedhomeip/chip-build-esp32-qemu:${VERSION}
-COPY --from=nrf /var/nRF5_tools /var/nRF5_tools
-COPY --from=nrf /var/nRF5_SDK_for_Thread_and_Zigbee /var/nRF5_SDK_for_Thread_and_Zigbee
-COPY --from=nrf /var/gcc-arm-none-eabi-9-2019-q4-major /var/gcc-arm-none-eabi-9-2019-q4-major
-ENV NRF5_SDK_ROOT=/var/nRF5_SDK_for_Thread_and_Zigbee
-ENV NRF5_TOOLS_ROOT=/var/nRF5_tools
-ENV ARM_GCC_INSTALL_ROOT=/var/gcc-arm-none-eabi-9-2019-q4-major/bin/
+COPY --from=nrf /var/nRF5_tools /opt/nrf5_tools
```

Comment:
what's the `/var` vs `/opt` rationale?  should the base images be using `/opt`?

Reply:
It doesn't really matter, but there's actually a spec for this, and /var isn't intended for software installation.

This snowballed from /sdk_support being too generic for the shared devcontainer into moving everything in to where it should be.

Reply:
Updated the base images. Adopted a convention of sorts: /opt paths match github.